README.md

```diff
@@ -18,11 +36,19 @@ namespace ConsoleApp1
         static void Main(string[] args)
         {
             var services = new ServiceCollection();
-            services.AddSingleton<ICommandHandler, MoveCommandHandler>();
-            services.AddCLI();
+            services.AddLogging(builder =>
+            {
+                builder.AddConsole();
+                builder.AddDebug();
+            });
+            services.AddCLI(config => {
+                config.Map<MoveCommand, MoveCommandHandler>()
+                      .MapHelpCommand();
```

Comment:
As I understand you can use default implementation of `help` command handler. My thought flow here was:
* what if I want to register other handler for help?
* maybe help command should be an interface that you can implement yourself?
* oh, but you may just don't you `MapHelpCommand` and just write you own class for this
* so eventually I'd call this extension method `MapDefaultHelpCommand` and it would be fine